purchase_request_department/views/purchase_request_department_view.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This line can be removed for ascetics, it hasn't been needed since v11, not a stopper though.